tests/cmdline/commands/test_devel.py

```diff
@@ -56,3 +56,38 @@ def test_launch_add_code(run_cli_command, aiida_code_installed):
 
     node = QueryBuilder().append(ProcessNode, tag='node').order_by({'node': {'ctime': 'desc'}}).first(flat=True)
     assert node.is_finished_ok
+
+
+def test_launch_multiply_add(run_cli_command):
+    """Test ``verdi devel launch-multiply-add``.
+
+    Start with a clean profile such that the functionality that automatically sets up the localhost computer and code is
+    tested properly.
+    """
+    result = run_cli_command(cmd_devel.devel_launch_multiply_add)
+    assert re.search(r'Warning: No `localhost` computer exists yet: creating and configuring', result.stdout)
+    assert re.search(r'Success: MultiplyAddWorkChain<.*> finished successfully', result.stdout)
+
+    node = QueryBuilder().append(WorkChainNode, tag='node').order_by({'node': {'ctime': 'desc'}}).first(flat=True)
+    assert node.is_finished_ok
+
+
+@pytest.mark.usefixtures('started_daemon_client')
+def test_launch_multiply_add_daemon(run_cli_command, submit_and_await):
+    """Test ``verdi devel launch-multiply-add`` with the ``--daemon`` flag."""
+    result = run_cli_command(cmd_devel.devel_launch_multiply_add, ['--daemon'])
+    assert re.search(r'Submitted workflow', result.stdout)
+
+    node = QueryBuilder().append(ProcessNode, tag='node').order_by({'node': {'ctime': 'desc'}}).first(flat=True)
+    submit_and_await(node)
```

Comment:
very useful command, did not know about it

Reply:
Literally copy-pasted everything from the tests for `verdi devel launch-add` :D